plugins/out_es/es.c

```diff
@@ -886,12 +886,13 @@ static void cb_es_flush(struct flb_event_chunk *event_chunk,
                 if (ctx->trace_error) {
                     /*
                      * If trace_error is set, trace the actual
-                     * input/output to Elasticsearch that caused the problem.
+                     * response from Elasticsearch explaining the problem.
+                     * Trace_Output can be used to see the request. 
+                     * We must use fwrite since the flb_log functions
+                     * will truncate data. 
                      */
-                    flb_plg_debug(ctx->ins, "error caused by: Input\n%s\n",
-                                  pack);
-                    flb_plg_error(ctx->ins, "error: Output\n%s",
-                                  c->resp.payload);
+                    fwrite(c->resp.payload, 1, c->resp.payload_size, stdout);
```

Comment:
This patch changes log format.

How about checking payload_size ?
If `c->resp.payload_size` is less than 4k (truncated size), use flb_plg_* to keep same format if the log will not be truncated.

Reply:
Sure I can do that

Reply:
Fixed